<?xml version="1.0"?>
<launch>
<!-- Source: https://github.com/Pallav1299/Navigate_Multiple_Robots -->
  <!-- No namespace here as we will share this description.
       Access with slash at the beginning -->
  <param name="robot_description"
    command="$(find xacro)/xacro --inorder $(find autonomous_robots)/turtlebot3_description/urdf/turtlebot3_burger.urdf.xacro" />
   

  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find autonomous_robots)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -0.5 -y 0.5 -z 0" />
      <arg name="robot_name"  value="Robot1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find autonomous_robots)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -0.5 -y 1 -z 0" />
      <arg name="robot_name"  value="Robot2" />
    </include>
  </group>

  <!-- BEGIN ROBOT 3-->
  <group ns="robot3">
    <param name="tf_prefix" value="robot3_tf" />
    <include file="$(find autonomous_robots)/launch/one_robot.launch" >
      <arg name="init_pose" value="-x -0.5 -y 1.5 -z 0" />
      <arg name="robot_name"  value="Robot3" />
    </include>
  </group>
  
  <group ns="drone">
    <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" />
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/> 
  </group>

</launch>
